custom_components/tahoma/__init__.py

```diff
@@ -147,7 +147,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
         if platform:
             entities[platform].append(device)
             _LOGGER.debug(
-                "Added TaHoma device (%s - %s - %s - %s)",
+                "Added device (%s - %s - %s - %s)",
```

Comment:
TaHoma is already in the log, before the message. This applies to all changes here.